examples/rpl_udp/udp.c

```diff
@@ -93,9 +95,9 @@ static void *init_udp_server(void *arg)
 }
 
 /* UDP send command */
-void udp_send(int argc, char **argv)
+int udp_send(int argc, char **argv)
 {
-    int sock;
+    int sock, res;
```

Comment:
Sorry, my mistake https://github.com/RIOT-OS/RIOT/pull/2602/files#diff-0de396c2e62be3e1c65c4f289e9f45f1R143 and https://github.com/RIOT-OS/RIOT/pull/2602/files#diff-0de396c2e62be3e1c65c4f289e9f45f1R149